proto/kvrpcpb.proto

```diff
@@ -858,3 +858,9 @@ message KeyRange {
     bytes start_key = 1;
     bytes end_key = 2;
 }
+
+enum ExtraRead {
```

Comment:
Why is this defined in `kvrpcpb.proto`?

Reply:
Both `storage` and `raftstore` will use it.

Reply:
It's okay to put them in cdcpb as cdcpb does not depend on both `raftstore` and `storage`.

Reply:
But it's a switch which could control the operation in `txn`, so I guess it's a little strange to put it to cdcpb.🤔